gci/views.py

```diff
@@ -66,4 +81,4 @@ def index(request):
     s.append('<script src="static/timeago.js"></script>')
     s.append('<script>loadTimeElements()</script>')
 
-    return HttpResponse('\n'.join(s))
+    return s
```

Comment:
Did it work without `.join` ?

Reply:
I dont understand.

Reply:
`s.append` is a list operation, not a string operation.

Reply:
NVM, meant to say why you didn't do join,
but it's called in `index` apparently.
Sorry.